### DISCUSSION TITLE
Justfile cheat sheet on cheatography.com

### DISCUSSION THREAD

**Discussion by linux-china** [casey/just#1052] [Ideas]
Justfile cheat sheet,  and feedbacks are welcome.   https://cheatography.com/linux-china/cheat-sheets/justfile/ 

![image](https://user-images.githubusercontent.com/46711/147144380-a8f9ced4-0062-46cb-81d8-1fc08eaa1a28.png)

**Comment by casey** [comment]
This is sick! Definitely going to add a link to it in the readme.
